packages/contracts-bedrock/src/libraries/DevFeatures.sol

```diff
@@ -16,11 +16,11 @@ library DevFeatures {
 
     /// @notice The feature that enables deployment of the CANNON_KONA fault dispute game.
     /// This feature depends on the DEPLOY_V2_DISPUTE_GAMES feature
-    bytes32 public constant CANNON_KONA = bytes32(0x0000000000000000000000000000000000000000000000000000000000000010);
+    //bytes32 public constant CANNON_KONA = bytes32(0x0000000000000000000000000000000000000000000000000000000000000010);
```

Comment:
I think if we want to leave these here we should probably just leave them defined and add a legacy natspec tag or something.